assistant/v2.ts

```diff
@@ -196,11 +190,7 @@ class AssistantV2 extends BaseService {
       'include_audit': _params.includeAudit,
     };
 
-    const sdkHeaders = getSdkHeaders(
-      AssistantV2.DEFAULT_SERVICE_NAME,
-      'v2',
-      'listAssistants'
-    );
+    const sdkHeaders = getSdkHeaders(AssistantV2.DEFAULT_SERVICE_NAME, 'v2', 'listAssistants');
```

Comment:
are these indent changes a deliberate stylistic change or are we forgetting to run something like prettier?

Reply:
They are a deliberate stylistic change from the sdk-generator